bin/crontabScripts/autoUpdateRepo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Can commit the corn job as well in a new file.

Reply:
will add to README

Reply:
I think SharePoint might be a better place for now. We will move it into readme later.

Reply:
Will add it under the "Vanilla ELK SetUp" section for now